public/app/features/explore/Explore.test.tsx

```diff
@@ -75,6 +75,14 @@ const dummyProps: Props = {
         to: 'now',
       },
     },
+    graphFrames: [],
+    logsFrames: [],
+    tableFrames: [],
+    traceFrames: [],
+    nodeGraphFrames: [],
+    graphResult: null,
+    logsResult: null,
+    tableResult: null,
```

Comment:
this is needed because of the changes in https://github.com/grafana/grafana/pull/45148/files#diff-70e1cc4ad5a5754fe5e6e51db45d1be0501db91555cbfb6db1fc0f80aa328415R200